### DISCUSSION TITLE
Please improve mobile support

### DISCUSSION THREAD

**Discussion by denis-v-s** [st3v3nmw/obsidian-spaced-repetition#240] [Q&A] [answered]
This is one of the most useful plugins for obsidian, however the layout is a mess on mobile. Please consider improving it.

**Comment by st3v3nmw** [comment, marked as answer]
Is this the flashcard size issue as #231?

**Reply by denis-v-s** [reply]
Seems like that does the trick! Thank you for pointing it out.

**Reply by st3v3nmw** [reply]
Btw @haosmark, which mobile device are you on? I'm not too sure what's causing this.

**Reply by denis-v-s** [reply]
Galaxy S8+
